<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Account</class>
 <widget class="QDialog" name="Account">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>481</width>
     <height>293</height>
    </rect>
   </property>
   <layout class="QGridLayout">
    <item row="4" column="0">
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>Logfile:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>SIP password:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1" colspan="2">
     <widget class="QSpinBox" name="sipPortSpinBox">
      <property name="toolTip">
       <string>Set 0 to use a random SIP port!</string>
      </property>
      <property name="minimum">
       <number>1025</number>
      </property>
      <property name="maximum">
       <number>65535</number>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLineEdit" name="usernameEdit">
      <property name="toolTip">
       <string>username of your SIP address (without domain, unless this is explicitely required by the SIP service)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="logFilePushButton">
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="label_15">
      <property name="text">
       <string>Local SIP Port:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <widget class="QComboBox" name="transportComboBox">
      <property name="toolTip">
       <string>the choosen transport protocol for SIP; Note: If you choose an IPv6 transport you have to configure the outbound proxy with an IPv6 IP address</string>
      </property>
      <item>
       <property name="text">
        <string>UDP</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>TCP</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Protocol:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="label_14">
      <property name="text">
       <string>Log Level:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="logFileEdit">
      <property name="toolTip">
       <string>optional: if a file is chosen, the logging will also be written to the specified file</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="2">
     <widget class="QLineEdit" name="domainEdit">
      <property name="toolTip">
       <string>the domain of your SIP address (e.g. fwd.pulver.com or iptel.org)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLineEdit" name="passwordEdit">
      <property name="toolTip">
       <string>the password of your SIP account</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>SIP user name:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1" colspan="2">
     <widget class="QComboBox" name="logLevelComboBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="toolTip">
       <string>the choosen transport protocol for SIP</string>
      </property>
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>6</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>SIP domain:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>391</width>
     <height>34</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="_2">
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="publishBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>340</y>
     <width>194</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>if this checkbox is enabled, the SIP client sends PUBLISH requests to send its presence state to the presence server</string>
   </property>
   <property name="text">
    <string>PUBLISH presence state</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="subscribeBox">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>340</y>
     <width>168</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>if this checkbox is enabled, the SIP client SUBSCRIBE to the buddies presence</string>
   </property>
   <property name="text">
    <string>SUBSCRIBE buddies</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
